### DISCUSSION TITLE
design options for interactive video queue

### DISCUSSION THREAD

**Discussion by mueslimak3r** [jellyfin/jellyfin-androidtv#1288] [General]
Currently the video queue is only used during playback, for tv series and playlists.
There is an "add to queue" button for video items but it does nothing since selecting a video item and playing it will overwrite the queue.

I'd like to implement a system for the video queue that allows viewing the queue, starting playback after adding items to it, and potentially having most or all of the features of the audio queue (shuffle, repeat, etc?)

A simple way to make the video queue at least usable would be to transform the `nowPlayingBug` to display "play video queue" when appropriate.

**Comment by nielsvanvelzen** [comment]
I think this is something that should be deferred until after the playback rewrite (#1057). The rewrite already has a way better queue system that allows mixing both video/audio.
